src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -488,13 +488,13 @@ export default {
 			// If no selection, replace the whole data
 			if (!selection.rangeCount) {
 				this.updateValue(html)
+				return
 			}
 
 			// Generate text and insert
-			const text = this.parseContent(html)
 			const range = selection.getRangeAt(0)
 			selection.deleteFromDocument()
-			range.insertNode(document.createTextNode(text))
```

Comment:
No, this is a security issue.
Parsing the content ensure we are only rendering what we support. It's been a while, but I do remember this is very important to keep as is.